2주차/12789/11866_python_제로.py

```diff
@@ -0,0 +1,27 @@
+import sys
+from collections import deque
+
+n = int(sys.stdin.readline())
+dequeLst = deque(map(int, sys.stdin.readline().split())) # 왼쪽 뽑는 기본 deque
+checkLst = []  # deque에서 뽑힌 추가로 저장된 아래진영
+current = 1  # 시작 번호
+
+for i in range(n): # deque
+    left = dequeLst.popleft()
+    if left == current:
+        current += 1
+        for j in range(len(checkLst)): # 추가진영
```

Comment:
```suggestion
        for _ in range(len(checkLst)): # 추가진영
```
iterable 객체를 반복문에서 사용할 때 변수가 사용되지 않으면 언더스코어로 나타내는 것도 좋다고 생각합니다.